### ISSUE TITLE
New translation: Arabic

### ISSUE THREAD

**Issue by yacir** [squidfunk/mkdocs-material#700]
``` jinja
{% macro t(key) %}{{ {
  "language": "ar",
  "direction": "rtl",
  "clipboard.copy": "نسخ إلى الحافظة",
  "clipboard.copied": "تم النسخ الى الحافظة",
  "edit.link.title": "عدل الصفحة",
  "footer.previous": "السابقة",
  "footer.next": "التالية",
  "meta.comments": "التعليقات",
  "meta.source": "المصدر",
  "search.language": "ar",
  "search.pipeline.stopwords": false,
  "search.pipeline.trimmer": false,
  "search.placeholder": "بحث",
  "search.result.placeholder": "اكتب لبدء البحث",
  "search.result.none": "لا توجد نتائج",
  "search.result.one": "نتائج البحث مستند واحد",
  "search.result.other": "نتائج البحث # مستندات",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "انتقل إلى المحتوى",
  "source.link.title": "اذهب إلى المصدر",
  "toc.title": "جدول المحتويات"
}[key] }}{% endmacro %}
```

Hope it helps :)

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Released as part of 2.6.2.
